pom.xml

```diff
@@ -48,8 +48,7 @@
     <logback.version>1.2.3</logback.version>
     <spring-version>5.1.1.RELEASE</spring-version>
     <jetty-test-policy.version>1.2</jetty-test-policy.version>
-    <servlet.api.version>4.0.1</servlet.api.version>
-    <servlet.schema.version>4.0.3</servlet.schema.version>
+    <servlet.api.version>4.0.1-SNAPSHOT</servlet.api.version>
```

Comment:
SNAPSHOT? Shouldn't it be a released version?

Reply:
The jetty-servlet-api jar in testing at the moment.
Close to being a release.
But yes, it should be a release soon.